dashboard/http_server_head.py

```diff
@@ -146,6 +146,19 @@ async def path_clean_middleware(self, request, handler):
                 raise aiohttp.web.HTTPForbidden()
         return await handler(request)
 
+    @aiohttp.web.middleware
+    async def browsers_no_post_put_middleware(self, request, handler):
+        if (
+            request.path.startswith("/api")
```

Comment:
I think this path prefix is technically unnecessary

Reply:
Also, I'm not sure how `aiohttp` handles casing -- should we do `request.path.lower()`?

Reply:
yeah you're right. We should allow POST or PUTS anywhere. will adjust